### ISSUE TITLE
provide sigla.json

### ISSUE THREAD

**Issue by flicksolutions** [DHBern/parzival-static-api#1]
Please provide a complete JSON of all sigils so I can replace the static file in the presentation repo.

This is the current file: https://github.com/DHBern/presentation_parzival/blob/main/src/lib/sigla.json

Makes sense to have this in the static api, right?

**Comment by pdaengeli** [comment]
Good point. I started preparing this on 25th Feb but never got round to integrate it into the pipeline (and the definition where to get the value for `sigil` is to be done, too).

Will do it when back in the Parzival context and then ping you.

**Comment by pdaengeli** [comment]
@flicksolutions, I just pushed the data to https://dhbern.github.io/parzival-static-api/api/json/metadata-nomenclature.json ([source](https://github.com/DHBern/parzival-static-api/blob/master/src/util/extract-fragment-metadata.xsl)).

The denominations of the fragments are not part of the data, so I scraped them from https://parzival.unibe.ch/hsverz.html.

Many names contain superscript characters. Especially in the case of Greek not all of them are part of Unicode. Right now there is a mix of superscript variants and normal variants:

```json
{ "sigil":"Fr. 44 (Gᵋᵋ)",  <--
  "loc":"Wilhelmsstift", 
  "aka":"",
  "cod":"Cod. Gb 676",
  "handle":"fr44"
},
{ "sigil":"Fr. 45 (Gζ)",   <--
  "loc":"\"Stadtarchiv\" \"Landschaftsverband Westfalen-Lippe-Museum für Kunst und Kultur, Westfälisches Landesmuseum\"",
  "aka":"\"ohne  signatur\", \"Ms. 459\"",
  "cod":"fr45",
  "handle":"fr45"
}
```

I opted for this because the specification recommends the textual variants (as opposed to markup) for semantic distinctions. But I understand that mixing the two might look bad in the frontend.

It would be possible (and more consistent) to do it the other way and include HTML `sup` tags in the values, e.g. `"sigil":"Fr. 44 (G<sup>ᵋᵋ</sup>)",` and `sigil":"Fr. 45 (G<sup>ζ</sup>)",`. Is that preferable?

**Comment by flicksolutions** [comment]
I'm baffled by those naming decisions again and again...
Thank you for digging into this!
Including HTML in the values sounds very good; there is a catch: Only the non-superscript characters should be wrapped in `<sup>`. `<sup>ᵋᵋ</sup>` -> would turn out to be doubly superscripted.
So, if there is an easy way to produce an output with `<sup>`-wrapped non-superscript characters, that would be the cleanest. If not, use all non-superscript characters and wrap them with `<sup>` - I would say.

**Comment by flicksolutions** [comment]
perfect!!
